File: HappyWayApp/ClientApp/src/app/models/event.model.ts
import {EventPlaceViewModel} from "./event-place.model";

export class EventViewModel {
  id: number;
  name: string;
  date: Date;
  user: string;
  completed: boolean;
  eventPlaceId: number;
  eventPlace: EventPlaceViewModel;

  constructor(event: EventModel, eventPlaceVm: EventPlaceViewModel) {
    this.id = event.id;
    this.name = event.name;
    this.date = event.date;
    this.user = event.user;
    this.completed = event.completed;
    this.eventPlaceId = event.eventPlaceId;
    this.eventPlace = eventPlaceVm;
  }
}

export class EventModel {
  id: number;
  name: string;
  date: Date;
  user: string;
  completed: boolean;
  eventPlaceId: number;
}

File: HappyWayApp/ViewModels/Response/EventModel.cs
using System;

namespace HappyWayApp.ViewModels.Response
{
    public class EventModel
    {
        public int Id { get; set; }

        public DateTime Date { get; set; }

        public string Name { get; set; }

        public bool Completed { get; set; }

        public int EventPlaceId { get; set; }

        public string User { get; set; }
    }
}
